stdlib/3/builtins.pyi

```diff
@@ -937,11 +937,28 @@ class SystemExit(BaseException):
     code = 0
 class Exception(BaseException): ...
 class ArithmeticError(Exception): ...
-class EnvironmentError(Exception):
-    errno = 0
-    strerror = ...  # type: str
-    # TODO can this be bytes?
-    filename = ...  # type: str
+if sys.version_info >= (3, 3):
+    class OSError(Exception):
+        errno = 0
+        strerror = ...  # type: str
+        filename = ...  # type: Union[str, bytes, None]
+        if sys.version_info >= (3, 4):
+            filename2 = ...  # type: Union[str, bytes, None]
+        if sys.platform == 'win32':
+            winerror = 0
```

Comment:
Is it a good idea to put `sys.platform` checks in? The `winerror` attribute is missing entirely on non-Windows platforms.